test/test_common_arcana.rb

```diff
@@ -10,19 +9,28 @@ class TestDRCA < Minitest::Test
 
   def setup
     reset_data
+    load_base_settings
   end
 
   def teardown
     @test.join if @test
   end
 
+  def load_base_settings(settings = {})
```

Comment:
Should this have gone in the base harness?

Reply:
Good point, I’ll move it. There’s a couple other tests where this method or similar exist in them too, I’ll evaluate refactoring them. Thanks